servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/FlushStrategyOnServerTest.java

```diff
@@ -79,8 +67,6 @@ class FlushStrategyOnServerTest {
     private static final String USE_EMPTY_RESP_BODY = "empty-resp-body";
 
     private OutboundWriteEventsInterceptor interceptor;
-    private HttpHeadersFactory headersFactory;
-
     private ServerContext serverContext;
```

Comment:
nit: you added a non null check in `tearDown`, should these be marked as Nullable to make IJ happier?